package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -161,6 +161,12 @@ class NucPairDist(AnalysisBase):
         helper for selecting atom pairs for distance analysis.
     """
 
+    _analysis_algorithm_is_parallelizable = True
+
+    @classmethod
+    def get_supported_backends(cls):
+        return ('serial', 'multiprocessing', 'dask',)
+    
     _s1: mda.AtomGroup
     _s2: mda.AtomGroup
     _n_sel: int
```

Comment:
Not related to this PR, but I don't think `_res_dict` (the line below) is used anymore in the code.

Reply:
Adjusted the PEP8 issues, should we keep `_res_dict` for now?

Reply:
You can just remove it; don't think theres's any need to add notes, as it’s only used internally and for type hinting.

Reply:
Done, true makes sense, since it is not used in the code after that.

Reply:
okay removing `_res_dict` led to the pytests failing, I will bring it back again.

Reply:
nah my bad, it was some other failure not connected to this code, so it should be fine